### ISSUE TITLE
Applicant program details page is missing h1 heading

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3016]
Found while adding axe accessibility testing.

Details from axe: https://dequeuniversity.com/rules/axe/4.4/page-has-heading-one?application=axeAPI

We currently have a level 2 heading. We should either bump this up to a level 1 heading, or evaluate if it makes sense to add a separate level 1 heading.

Example: 
![Screen Shot 2022-08-01 at 12 29 48 PM](https://user-images.githubusercontent.com/9094240/182230225-1ccc757f-1dba-4c3a-b1b1-b961a93aaf7b.png)

**Comment by shanemc-goog** [comment]
Which Program details?

I'm fixing an issue with a H1 following a H2 on the screen when an Admin views applications.  I'm removing the H1 fwiw.

**Comment by jeanniefu** [comment]
Ah this is from the applicant view. Added more details and a screenshot to this issue.
